config/initializers/devise.rb

```diff
@@ -1,17 +1,3 @@
-class TurboFailureApp < Devise::FailureApp
```

Comment:
The changes in this file are taken from https://github.com/heartcombo/devise/issues/5560